### ISSUE TITLE
TTv5 Q&A: Broken link

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#171]
Location: Search results page
URL: https://dev.section508testing.net/mod/forum/search.php?id=89&words=&phrase=&notwords=&fullwords=&hfromday=1&hfrommonth=1&hfromyear=1&hfromhour=1&hfromminute=1&htoday=1&htomonth=1&htoyear=1&htohour=1&htominute=1&forumid=40&subject=&user=
Issue: the 'accessibility@hq.dhs.gov' link is not accessible (via keyboard or mouse).  The link needs to be made accessible.

This issue also resides on the Trusted Tester Q&A - June 18, 2019 > Post Questions Here page
URL: https://dev.section508testing.net/mod/forum/discuss.php?d=20.  Please correct the issue globally throughout the application.

![Accessibility Link (Search Results page)](https://user-images.githubusercontent.com/34749031/59946483-a2af7d00-9438-11e9-9ef7-2b26341e96b2.jpg)

**Comment by drewnielson** [comment]
Added link. For future reference, @HD508Github, if something is neither accessible by mouse or keyboard, then this is not considered a 508 failure. It is quite possible to provide an email address without providing it as a link.
